charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -42,6 +42,15 @@ privilegedAccess:
   allowedAccessIDs: []
    # - p-1234
    # - p-5678
+  service_account:
```

Comment:
use "serviceAccoun" instead, it's Helm values convention to use camel case and one-word attributes without "_" lowercases